src/controllers/expense.controller.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const expenseService = require('../services/expenses.service');
+const { getUserById } = require('../services/user.service');
+
+const getExpenses = async(req, res) => {
```

Comment:
controllers are just a middleware between route and service and they r supposed to have as least logic as possible, so pls consider moveing the majority of logic to services